pages/Home.vue

```diff
@@ -198,6 +211,17 @@ export default {
   },
   mounted () {
     if (!this.isLoggedIn && localStorage.getItem('redirect')) { this.$bus.$emit('modal-show', 'modal-signup'); }
+    this.banners = this.promotedOffers.mainBanners
+    this.heroes = this.heroImages
+    supportsWebP.then(supported => {
+      this.createBanners(supported)
+      this.createHeroes(supported)
```

Comment:
what you think about moving all this logic to computed? if data will change then we will have problem to recreate this data and computed can handle that for us

Reply:
good idea